javascript/src/seedsource/containers/Upload.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import shp from 'shpjs'
```

Comment:
Make sure to add `shpjs` to `package.json`.

Reply:
In parent repository right?

Reply:
Right.

Reply:
Ah, I see you've already done this. Never mind!